src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2308,6 +2322,16 @@ private void processFileChanges(
                     // re-enable debug variables in server.env
                     enableServerDebug(false);
                 }
+                if (container && System.getProperty("os.name").equalsIgnoreCase("linux")) {
+                    info("Restarting the container is required on Linux for it to take effect.");
```

Comment:
This should be rephrased, since it currently sounds as if it's asking the user to restart the container, when in fact it is doing it automatically.

Reply:
👍